Cannot create any bill in a project without currency

- create a project with no currency
- add participants
- try to add a bill: it fails with:

```
Currency: Not a valid choice
```

I suspect that the form contains an actual currency, even though it isn't shown in the UI.

I'm surprised that the tests don't catch this kind of big issue...

Ok, tests fail as expected (with sometimes hard-to-understand errors, but it's minor):

```
api_test.py::APITestCase::test_bills - AssertionError: 201 != 400 :  expected 201, got 400
api_test.py::APITestCase::test_bills_with_calculation - AssertionError: 201 != 400 :  expected 201, got 400
api_test.py::APITestCase::test_statistics - AssertionError: Lists differ: [{'balance': 12.5, 'member': {'activated': T[190 chars]2.5}] != [{'member': {'id': 1, 'name': 'zorglub', 'we[1...
api_test.py::APITestCase::test_weighted_bills - TypeError: string indices must be integers
budget_test.py::BudgetTestCase::test_access_other_projects - AssertionError: 0 != 1
budget_test.py::BudgetTestCase::test_export - TypeError: send_file() got an unexpected keyword argument 'download_name'
budget_test.py::BudgetTestCase::test_manage_bills - sqlalchemy.orm.exc.NoResultFound: No row was found for one()
budget_test.py::BudgetTestCase::test_membership - AssertionError: 1 != 2
budget_test.py::BudgetTestCase::test_person_model - AssertionError: False is not true
budget_test.py::BudgetTestCase::test_rounding - AssertionError: 0 != 8.12
budget_test.py::BudgetTestCase::test_statistics - AssertionError: Regex didn't match: '<td class=\\"d-md-none\\">zorglub</td>\\s+<td>\\$20\\.00</td>\\s+<td>\\$31\\.67</td>' not found i...
budget_test.py::BudgetTestCase::test_weighted_balance - AssertionError: Items in the first set but not the second:
history_test.py::HistoryTestCase::test_bill_add_remove_add - AssertionError: 3 != 5
history_test.py::HistoryTestCase::test_clear_ip_records - sqlalchemy.orm.exc.NoResultFound: No row was found for one()
history_test.py::HistoryTestCase::test_disable_clear_no_new_records - sqlalchemy.orm.exc.NoResultFound: No row was found for one()
history_test.py::HistoryTestCase::test_double_bill_double_person_edit_second - AssertionError: 3 != 5
history_test.py::HistoryTestCase::test_logs_for_common_actions - AssertionError: 'Bill <em class="font-italic">fromage à raclette</em> added' not found
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -34,7 +34,7 @@ install_requires =
     Flask-RESTful~=0.3
     Flask-SQLAlchemy~=2.4
     Flask-WTF~=0.14,>=0.14.3  # See b76da172652da94c1f9c4b2fdd965375da8a2c3f
-    WTForms>=2.2.1,<2.4
+    WTForms>=2.3.1,<2.4
     Flask>=1.1,<3.0
     itsdangerous>=1.1,<3.0
     Jinja2>=3.0,<4.0

